testing/testcafe/tests/dataGrid/scrolling.ts

```diff
@@ -1287,3 +1287,66 @@ test('New virtual mode. Navigation to the last row if new row is added (T1069849
     });
   });
 });
+
+// T1152498
+['infinite', 'virtual'].forEach((scrollingMode) => {
+  test(`${scrollingMode} scrolling - the markup should be correct for continuous scrolling when there is a fixed column with cellTemplate (React)`, async (t) => {
+  // arrange
+    const dataGrid = new DataGrid('#container');
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    // act
+    await dataGrid.scrollTo({ y: 200 });
+    await t.wait(200);
+    await dataGrid.scrollTo({ y: 400 });
+
+    // assert
+    await t
+      .wait(500)
+      .expect(await takeScreenshot(`grid-${scrollingMode}-scrolling-T1152498.png`, '#container'))
```

Comment:
Question.

Maybe we can move last await to act part of the tests?
```suggestion
    await dataGrid.scrollTo({ y: 200 });
    await t.wait(200);
    await dataGrid.scrollTo({ y: 400 });
    await t.wait(200);

    // assert
    await t
      .expect(await takeScreenshot(`grid-${scrollingMode}-scrolling-T1152498.png`, '#container'))
```